news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Arrival](https://osu.ppy.sh/u/1694000)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/next-level.jpg)](https://osu.ppy.sh/s/623246)
+
+Winning best marathon and electronic map as well as capturing runner-up for Grand Award, **[MMzz](https://osu.ppy.sh/u/128993)**'s map of **[NEXT LEVEL](https://osu.ppy.sh/s/623246)** has universally won the respect of taiko mappers and players alike. One only has to play the map once to understand why -- **[MMzz](https://osu.ppy.sh/u/128993)** captures all the diversity offered by sky_delta and lapix's composition and transforms it into a cohesive playing experience. The relatively lower BPM of the map allows for liberal use of mixed-snapping patterns that play smoothly, keeping the map engaging from start to end.
+
+*written by [hikikochan](https://osu.ppy.sh/u/6512678)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/matsuri.jpg)](https://osu.ppy.sh/s/511420)
+
+A well rounded map completely deserving this place on the best spread of 2017. The song's characteristics gave the possibility of creating exotic lower difficulties using blue ticks efficiently. From Kantan to Inner Oni, these rhythmical particularities are well represented, giving both new and seasoned players a good challenge. One of the sets I'd recommend to new players starting taiko, the song and difficulties really sets it apart from the other ranked maps.
+
+*written by [Arrival](https://osu.ppy.sh/u/1694000)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/maniera.jpg)](https://osu.ppy.sh/s/400761)
+
+What makes **[MANIERA]( https://osu.ppy.sh/s/400761)** the most original map of 2017 you ask? Well it’s not about SV changes, hitsounds or storyboard, it’s about every single snapping and patterning choices that **[yu68]( https://osu.ppy.sh/u/6170507)** and his guests have made to make this map enjoyable to play. You don’t need to listen to this song twice to notice how complicated it is to interpret in mapping as it contains many different rhythms, with some that would make the map unplayable if followed accurately. What makes this set even more interesting is that each guest difficulty provides a unique style, resulting in a map that can please any sort of player!
+
+*written by [Nofool](https://osu.ppy.sh/u/672430)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/buriki.jpg)](https://osu.ppy.sh/s/482486)
+
+**[yu68]( https://osu.ppy.sh/u/6170507)** made his debut in the ranked category in 2017 with his map of **[SymaG's Buriki no Dance](https://osu.ppy.sh/s/482486)**, an energetic utaite cover of a Vocaloid song. The mapset is impressive not only as a first ranked map but also in general -- the spread is made cohesively and the top difficulty flows perfectly, making for a great map to play while jamming along to the music!
```

Comment:
```diff
- *[yu68]( https://osu.ppy
+ *[\_yu68](https://osu.ppy
```
missing underscore and omit space between `(` and `http`

Reply:
missing the underscore in other references to his name as well